quickstart/speech-translation/cpp-windows/helloworld.sln

```diff
@@ -1,4 +1,4 @@
-﻿
+
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
what got changed here? just the whitespace?

Reply:
there was a stray BOM